src/bindings/python/src/pyopenvino/core/common.cpp

```diff
@@ -20,17 +20,18 @@ namespace type_helpers {
 
 const std::map<ov::element::Type, py::dtype>& ov_type_to_dtype() {
     static const std::map<ov::element::Type, py::dtype> ov_type_to_dtype_mapping = {
-        {ov::element::f16, py::dtype("float16")},  {ov::element::bf16, py::dtype("float16")},
-        {ov::element::f32, py::dtype("float32")},  {ov::element::f64, py::dtype("float64")},
-        {ov::element::i8, py::dtype("int8")},      {ov::element::i16, py::dtype("int16")},
-        {ov::element::i32, py::dtype("int32")},    {ov::element::i64, py::dtype("int64")},
-        {ov::element::u8, py::dtype("uint8")},     {ov::element::u16, py::dtype("uint16")},
-        {ov::element::u32, py::dtype("uint32")},   {ov::element::u64, py::dtype("uint64")},
-        {ov::element::boolean, py::dtype("bool")}, {ov::element::u1, py::dtype("uint8")},
-        {ov::element::u4, py::dtype("uint8")},     {ov::element::nf4, py::dtype("uint8")},
-        {ov::element::i4, py::dtype("int8")},      {ov::element::f8e4m3, py::dtype("uint8")},
-        {ov::element::f8e5m2, py::dtype("uint8")}, {ov::element::string, py::dtype("bytes_")},
-        {ov::element::f4e2m1, py::dtype("uint8")}, {ov::element::f8e8m0, py::dtype("uint8")},
+        {ov::element::f16, py::dtype("float16")},   {ov::element::bf16, py::dtype("float16")},
+        {ov::element::f32, py::dtype("float32")},   {ov::element::f64, py::dtype("float64")},
+        {ov::element::i8, py::dtype("int8")},       {ov::element::i16, py::dtype("int16")},
+        {ov::element::i32, py::dtype("int32")},     {ov::element::i64, py::dtype("int64")},
+        {ov::element::u8, py::dtype("uint8")},      {ov::element::u16, py::dtype("uint16")},
+        {ov::element::u32, py::dtype("uint32")},    {ov::element::u64, py::dtype("uint64")},
+        {ov::element::boolean, py::dtype("bool")},  {ov::element::u1, py::dtype("uint8")},
```

Comment:
Please also take a look at `_get_strides()`: https://github.com/openvinotoolkit/openvino/blob/2c204f445e9faf66f103061f58944affa061f4b6/src/bindings/python/src/pyopenvino/core/common.cpp#L401-L403
How this method should behave for `u2`?